### ISSUE TITLE
Is it possible to crawl job postings?

### ISSUE THREAD

**Issue by yidingww** [linkedtales/scrapedin-linkedin-crawler#34]
Right now the crawler is only able to crawl the user profiles. Is it possible to use it to crawl the search result of jobs and their postings?

**Comment by leonardiwagner** [comment]
Yes, it is. Please collaborate sending the code to this project, thank you.
